_summaries/wu2018group_1.md

```diff
@@ -1,19 +1,107 @@
+normalization only differ in choosing the set $S_i$, the set of pixels that the
+mean and standard deviation are computed over. 
+
+* Batch normalization chooses the set of pixels that belong to the same channel
+index
+
+* Layer normalization chooses the set of pixels that belong to the same image
+
+* Instance normalization chooses the set of pixels that belong to the same
+channel and image
+
+* Group normalization chooses a group of channels in a single image.
+
+<img width="500px" src="wu2018group_1a.PNG"/>
+
+All four normalization methods learn the the variables $\gamma$ and $\beta$ for
+the formula
+
+$$
+y_i = \gamma \hat x_i + \beta
+$$
+
+while training for use in inference tasks.
+
+
+
+### How well does the paper perform?
+
+The authors of the paper ran several experiments over a variety of computer
+vision tasks. In general, batch normalization tended to slightly outperform
+group normalization with larger batch sizes, but tended to lost out to group
+normalization when the batch sizes became extremely small. 
+
+The first experiment in the paper compared the effect the batch size and
+different normalization methods had on accuracy for a neural network model
+training on the **ImageNet** dataset.
+
+<img width="500px" src="wu2018group_1c.PNG"/>
+
+Batch normalization performed the best with a batch size of 32 images per GPU
+(8 GPUs were used) outperforming group normalization by 0.5% on the validation
+set. However, group normalization achieved a 10.6% lower error rate with a batch
+size of 2, showing its effectiveness on lower batch sizes, The effects of group
+size on accuracy were also tested, the authors finding that a fixed size of 32
+groups are a variable size of 16 channels per group perfomed the best for their
+models.
+
+The authors also experimented with object detection and segmentation on the
+**COCO** dataset. Object detection and segmentation tend to involve higher
+resolution images, meaning smaller batch sizes tend to be used for these tasks.
+With 1 image per GPU for 8 GPUs being used for this task, group normalization
+outperformed batch normalization across the board.
+
+<img width="500px" src="wu2018group_1d.PNG"/>
+
+The final set of experiments in the paper had to do with video classification
+using the **Kinetics** dataset.There experiments covered 8 and 4 clips per GPU
+for 32-frame clips and 4 clips per GPU for 64-frame clips. Here, batch
+normalization outperformed group normalization for the 32 images per GPU batch
+size, but perfomed worse than group normalization when a batch size of 4 was
+used.
+
+<img width="500px" src="wu2018group_1e.PNG"/>
+
+
+
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Group Normalization outperforms Batch Normalization for small batch sizes
+* Group Normalization standardizes small groups of channels for each image
+* Group Normalization sufferes performance degradation compared to Batch
```

Comment:
What do you mean here? where do they see performance degradation in GN?

Reply:
Changed it to say that "Group Normalization performs worse than Batch Normalization for larger batch sizes."